### ISSUE TITLE
Offer AI Example

### ISSUE THREAD

**Issue by roy100** [productml/blurr#10]
Write the first example

**Comment by mserranom** [comment]
We need a better description

**Comment by mserranom** [comment]
Hey Roy, here's a suggestion on how I'd expect a walkthrough from an engineer perspective:

1. The problem
    * We want to show personalised offers
    * We have raw events, but that's not enough, we need features to train a model
2. Step by step transformations
    * Simplest session aggregate (always with inputs and outputs)
    * Aggregating more data
    * Identities
    * Windows, etc.
3. Training with SageMaker
4. Serving the model
    * We only have user id, we can't query the model like that
    * accessing the prediction API

**Comment by roy100** [comment]
Nice format! 

1. We don't have a 'centralized readme' yet, so I wrote the offer AI example to start with a mini-readme so that we share just this example to Jose and Julian
2. I don't think we need a simple session aggregate example (maybe a separate one?) - when I saw Jose looking at the DTC, he seemed to straightaway get what the SessionAggregates piece was doing with zero effort
3. Serving the model is the most unclear part because 1) we don't know what prediction API the user will use and 2) our windowing implementation is currently unclear. I think this bit needs some discussion but I've stuck that flowchart in for the time being
